proposals/stage-1/PROJECT_ONBOARDING/NEW_PROJECT_APPLICATION.md

```diff
@@ -38,7 +38,7 @@ Full list of dependencies in your source tree including licenses.
 
 ## Release Methodology and mechanics
 
-How often do you release and where are the releases made. Do you have an LTS program?
```

Comment:
"an" should be "a" but not deleted

Reply:
Not sure what you mean.

Reply:
Sorry, I misread the diff reviewing on my phone. You can ignore this and I'll resolve this comment.